packages/n4s/src/rules/__tests__/isValueOf.test.ts

```diff
@@ -0,0 +1,36 @@
+import { enforce } from 'enforce';
+import { isValueOf, isNotValueOf } from 'isValueOf';
+
+const objectToTest = {
+  a: 'Alpha',
+  b: 'Bravo',
+  c: 'Charlie',
+};
+
```

Comment:
Let's add a few more test cases - for example, when the passed object is not really an object (`null`, `undefined`, ...etc)